src/c_kzg_4844.c

```diff
@@ -1743,9 +1671,9 @@ static C_KZG_RET new_fft_settings(FFTSettings *fs, unsigned int max_scale) {
  * @remark Free after use with free_trusted_setup().
  *
  * @param[out] out      Pointer to the stored trusted setup data
- * @param[in]  g1_bytes Array of G1 points
+ * @param[in]  g1_bytes Array of G1 points in Lagrange form
  * @param[in]  n1       Number of `g1` points in g1_bytes
- * @param[in]  g2_bytes Array of G2 points
+ * @param[in]  g2_bytes Array of G2 points in monomial form
```

Comment:
Is it really called monomial form? I don't have a strong name for this, but I have some options.

1. coefficient form?
2. normal form?

I think I would go for "normal form". The monomial one is quite confusing to me.

Reply:
That's what @kevaundray refers to them in go-kzg. For example: [this search](https://github.com/search?q=repo%3Acrate-crypto%2Fgo-kzg-4844%20monomial&type=code).

I'm not sure if it's the most accurate name. @asn-d6 @dankrad thoughts?

Reply:
IMO either monomial form or coefficient form is fine in this context.